pyEpiabm/pyEpiabm/tests/test_household_sweep.py

```diff
@@ -35,16 +43,17 @@ def test__call__(self, mock_inf, mock_susc):
         """
         mock_inf.return_value = 10
         mock_susc.return_value = 10
```

Comment:
```suggestion
        mock_inf.return_value = 10.0
        mock_susc.return_value = 10.0
```

Tiny point so feel free to ignore, but I think these both return floats